sdk/eventhub/azure-eventhub/stress/scripts/dev_requirement.txt

```diff
@@ -1,9 +1,10 @@
 psutil
-azure-eventhub==5.8.0a5
+azure-eventhub
 azure-eventhub-checkpointstoreblob
 azure-eventhub-checkpointstoreblob-aio
 azure-servicebus==0.50.3
```

Comment:
isnt this a really old version ?

Reply:
yup I was going to just leave this as azure-eventhub because I was thinking that the GA of pyamqp EH would be the new version -- can also make it the beta version, but either way this will have to be updated in the very near future/ for any local testing

Reply:
sorry I meant servicebus :)

Reply:
ah yes it can get bumped